.github/workflows/release.yaml

```diff
@@ -165,44 +165,44 @@ jobs:
         else
           OPERATOR_QUAY_REPO="${{ github.event.inputs.operator_quay_repository }}"
         fi
-        
+
         PLUGIN_QUAY_TAG="$PLUGIN_QUAY_REPO:$RELEASE_VERSION"
         OPERATOR_QUAY_TAG="$OPERATOR_QUAY_REPO:$RELEASE_VERSION"
-        
-        echo "::set-output name=plugin_quay_tag::$PLUGIN_QUAY_TAG"
-        echo "::set-output name=plugin_quay_repo::$PLUGIN_QUAY_REPO"
-        echo "::set-output name=operator_quay_tag::$OPERATOR_QUAY_TAG"
-        echo "::set-output name=operator_quay_repo::$OPERATOR_QUAY_REPO"
-    
+
+        echo "plugin_quay_tag=$PLUGIN_QUAY_TAG" >> $GITHUB_ENV
+        echo "plugin_quay_repo=$PLUGIN_QUAY_REPO" >> $GITHUB_ENV
+        echo "operator_quay_tag=$OPERATOR_QUAY_TAG" >> $GITHUB_ENV
+        echo "operator_quay_repo=$OPERATOR_QUAY_REPO" >> $GITHUB_ENV
+
     - name: Cleanup
       run: rm bump.py minor.py
 
     - name: Log information
       run: |
-        echo "Release type: ${{ steps.release_type.outputs.release_type }}"
+        echo "Release type: ${{ env.release_type }}"
 
-        echo "Release version: ${{ steps.release_version.outputs.release_version }}"
+        echo "Release version: ${{ env.release_version }}"
 
-        echo "Next version: ${{ steps.next_version.outputs.next_version }}"
+        echo "Next version: ${{ env.next_version }}"
 
-        echo "Branch version: ${{ steps.branch_version.outputs.branch_version }}"
+        echo "Branch version: ${{ env.branch_version }}"
 
-        echo "Plugin Quay tag": ${{ steps.quay_tag.outputs.plugin_quay_tag }}
+        echo "Plugin Quay tag: ${{ env.plugin_quay_tag }}"
 
-        echo "Operator Quay tag": ${{ steps.quay_tag.outputs.operator_quay_tag }}
+        echo "Operator Quay tag: ${{ env.operator_quay_tag }}"
 
   release:
     name: Release
-    if: ${{ needs.initialize.outputs.release_type != 'skip' && ((github.event_name == 'schedule' && github.repository == 'kiali/openshift-servicemesh-plugin') || github.event_name != 'schedule') }}
```

Comment:
I'm not sure if `needs.initialize.output` can be replaced with env. @leandroberetta ?

Reply:
nm.. I need to keep this the way it was